config/event_handling.tmpl.yml

```diff
@@ -1444,6 +1454,18 @@ event_handling:
   NODE_NODENOTREADY:
   - refresh:
     - ems
+  NODE_NODESCHEDULABLE:
```

Comment:
@lfu as @simon3z asked https://github.com/ManageIQ/manageiq/pull/5155#issuecomment-151781047, will it be in 4.0?